src/object.cpp

```diff
@@ -95,7 +106,7 @@ ddwaf_object *ddwaf_object_stringl_nc(ddwaf_object *object, const char *string,
     return object;
 }
 
-ddwaf_object *ddwaf_object_signed(ddwaf_object *object, int64_t value)
+ddwaf_object *ddwaf_object_string_from_signed(ddwaf_object *object, int64_t value)
```

Comment:
NP: I like this new naming `from` but it feels a bit inconsistent

Reply:
How so?